src/commands/createVirtualMachine/VirtualNetworkCreateStep.ts

```diff
@@ -20,7 +20,9 @@ export class VirtualNetworkCreateStep extends AzureWizardExecuteStep<IVirtualMac
 
         const virtualNetworkProps: NetworkManagementModels.VirtualNetwork = { location, addressSpace: { addressPrefixes: [nonNullProp(context, 'addressPrefix')] } };
         const rgName: string = nonNullValueAndProp(context.resourceGroup, 'name');
-        const vnName: string = nonNullProp(context, 'newVirtualMachineName');
+        const vmName: string = nonNullProp(context, 'newVirtualMachineName');
```

Comment:
I'm not gonna say how long it took me to figure out what changed on this line 😂

Reply:
|\| |\/| !?!?!?